xcode/Subconscious/SubconsciousTests/Tests_NotebookUpdate.swift

```diff
@@ -76,7 +76,7 @@ class Tests_NotebookUpdate: XCTestCase {
     }
     
     func testViewerSlashlinkConstruction() throws {
-        let model = NotebookModel()
+        let model = DetailStackModel()
```

Comment:
Let's move these DetailStack-specific tests to a separate `Tests_DetailStack` unit test